### PR TITLE
Add MinIO clarification to upgrade instructions

### PR DESCRIPTION
**Pull request by macraig**

### CODE REVIEW

File: doc/admin/updates/pure_docker.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ To upgrade, please perform the changes in the following diff:
 
 https://github.com/sourcegraph/deploy-sourcegraph-docker/commit/223c11dacffafb985c2d29b6c6a9b84bcc8255be
 
-This upgrade removes the `code intel bundle manager`. This service has been deprecated and all references to it have been removed. 
+This upgrade removes the `code intel bundle manager`. This service has been deprecated and all references to it have been removed. This upgrade also adds a MinIO container that doesn't require any custom configuration. You can find more detailed documentation in https://docs.sourcegraph.com/admin/external_services/object_storage.
```
Comment (efritz) [inline doc/admin/updates/pure_docker.md:24]:
```suggestion
This upgrade removes the `code intel bundle manager`. This service has been deprecated and all references to it have been removed.

This upgrade also adds a MinIO container that doesn't require any custom configuration. You can find more detailed documentation in https://docs.sourcegraph.com/admin/external_services/object_storage.

```

**Comment by slimsag** [comment]
Thanks for doing this! Also add this to doc/admin/updates/kubernetes.md please :)

**Comment by efritz** [comment]
Same thing should go in the k8s and compose deployments. cc @slimsag It looks like server deployments are a bit out of date. Should we bother adding text here yet?

**Comment by slimsag** [comment]
@efritz yeah I think just kubernetes.md/docker_compose.md/pure_docker.md - no need to update server as that is completely contained and for demo/non-production deployments.

**Review by efritz — APPROVED** [review]

**Comment by daxmc99** [comment]
Can this be merged?

**Comment by macraig** [comment]
@daxmc99 thanks for the reminder!

**Comment by daxmc99** [comment]
🙏 Thank you!

**Pull request merged**